tests/activation/test_xonsh_activation.py

```diff
@@ -0,0 +1,49 @@
+import subprocess
+import sys
+from os.path import join, normcase
+
+import pytest
+
+import virtualenv
+from tests.lib import need_executable
+
+XONSH_COMMAND = "xonsh.bat" if virtualenv.is_win else "xonsh"
+
+
+def need_powershell(fn):
+    return pytest.mark.powershell(
+        need_executable("xonsh", (XONSH_COMMAND, "--version"))(fn)
+    )
+
+
+def print_python_exe_path():
+    return "{} -c 'import sys; print(sys.executable)'".format(virtualenv.expected_exe)
+
+
+@need_powershell
+def test_activate_with_powershell(tmpdir, monkeypatch):
+    monkeypatch.chdir(tmpdir)
```

Comment:
yeah, drop all powershell references

Reply:
Uggh Thanks for catching this.  I think I have gotten them all now.